File: app/src/main/java/com/gnardini/tvshowcontroller/networking/MouseService.java
package com.gnardini.tvshowcontroller.networking;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MouseService {

    @GET("/move_mouse")
    Call<String> moveMouse(@Query("x") float x, @Query("y") float y);

    @GET("/left_click")
    Call<String> leftClick();

    @GET("/right_click")
    Call<String> rightClick();

}

File: app/src/main/java/com/gnardini/tvshowcontroller/networking/MouseService.kt
package com.gnardini.tvshowcontroller.networking

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MouseService {

    @GET("/move_mouse")
    fun moveMouse(@Query("x") x: Float, @Query("y") y: Float): Call<String>

    @GET("/left_click")
    fun leftClick(): Call<String>

    @GET("/right_click")
    fun rightClick(): Call<String>

}
